.gitallowed

```diff
@@ -118,3 +118,8 @@ dockstore-webservice/src/main/java/io/swagger/model/ToolDescriptor.java:.*pcawg_
 README.md:.*https://github.com/dockstore/dockstore/blob/
 README.md:.*https://app.codacy.com/app/dockstore/dockstore
 README.md:.*db89e89508bd562cd36e9c316ac968ab0a136c0e
+
+# dockstore-webservice/src/main/java/io/dockstore/webservice/languages/LanguageHandlerInterface.java:
```

Comment:
these lines don't seem related to the other changes

Reply:
These were git-secrets issues that came up when I tried to make changes to the file, so I added ignores for them.

Reply:
Hmm, weird. Maybe someone overrode before